package.json

```diff
@@ -39,8 +39,6 @@
     "web-ext": "^4.1.0"
   },
   "scripts": {
-    "postversion": "git push https://github.com/hypothesis/browser-extension.git master:master --follow-tags",
-    "preversion": "npm run test",
```

Comment:
These steps were removed because the `Jenkinsfile` now handles creating the tag and pushing the version update commit back to the origin, as well as any pre-version steps.